config/01-ce-collector-audit-payload.conf

```diff
@@ -0,0 +1,24 @@
+# Settings for the collector audit payloads function.
+# The easiest way to direct the payload logging information here is to
+# set the environment variable CONDORCE_COLLECTOR_HOST on worker nodes
+# pointing to this condor-ce host.  Then pilot jobs that use condor
+# will automatically forward their payload classads here for auditing.
+
+# You can edit this file but that prevents future rpm upgrade
+# replacements, so it is better to override these settings in 
+# /etc/condor-ce/99-ce-collector-audit-payload.conf
```

Comment:
I understand, and this file is editable, but my comment is true because editing it means that the next upgrade will result in a .rpmnew and a little more hassle than putting changes into a new file.  I did forget the config.d in the path though, I'll add that.